.gitignore

```
@@ -32,5 +32,7 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
+/config/credentials/development.key
+/config/credentials/test.key
 /config/credentials/staging.key
 /config/credentials/production.key
```

```suggestion
/config/credentials/*.key
```
前回見逃していましたが、ここの場所の .key は全て ignore でいいかなと思いました！

---

こちら仰る通りですね。
普通に修正してしまいましたが、"Commit suggestion"でよかったですね……。